drivers/ieee802154/ieee802154_rf2xx_iface.c

```diff
@@ -248,3 +248,42 @@ void rf2xx_iface_frame_write(struct device *dev,
 	LOG_DBG("Frame W: PhyStatus: %02X. length: %02X", status, length);
 	LOG_HEXDUMP_DBG(data, length, "payload");
 }
+
+void rf2xx_iface_sram_read(struct device *dev,
+				  u8_t address,
```

Comment:
The alignment seems a bit off here and in the function declaration.

Reply:
Yes, this is annoying. I can't find a tool that I can do it on any GUI to programmatically fix that. I'm using VS-Code that show one thing, git diff shows another thing, gedit show another thing and github shows another things too and for sure no one of that tools are right.

I'll re-check is chackpath is enable on git commit.